rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -39,6 +39,12 @@ def _consume_rotki_trades(
         amount_sold = deserialize_asset_amount(csv_row['Sell Amount'])
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
         asset, fee, fee_currency, location, timestamp = process_rotki_generic_import_csv_fields(csv_row, 'Base Currency')  # noqa: E501
+        if csv_row['Type'].strip().lower() == 'buy':
+            trade_type = TradeType.BUY
+        elif csv_row['Type'].strip().lower() == 'sell':
+            trade_type = TradeType.SELL
```

Comment:
here if you need to got to the elif you are doing twice the `csv_row['Type'].strip().lower()`. I believe is better to create a variable (`sanitized_type`?) and store there `csv_row['Type'].strip().lower()` and then do the `if/else`

Reply:
I just noticed that we have a `deserialize`method in `TradeType`that we weren't using. Instead of comparing against `buy/sell` I would use that